doc/guides/modules.rst

```diff
@@ -487,6 +486,105 @@ variables. For example:
 
   include(${ZEPHYR_CURRENT_MODULE_DIR}/cmake/code.cmake)
 
+.. _modules_module_root:
+
+Module root
```

Comment:
need to change that from "Module root" to whatever module_ext_root means now.

Reply:
Done, including an update to `Build files placed in a module root`below.